frontend/src/pages/registration/registration.js

```diff
@@ -0,0 +1,128 @@
+import Button from '../../components/button/button';
+import {React, useState} from 'react';
+import Field from '../../components/field/field';
+import './registration.css'
+import personService from '../../services/person.service';
+import {useNavigate} from "react-router-dom";
+import validation from '../../validation';
+import {AiOutlineEyeInvisible, AiOutlineEye} from "react-icons/ai";
+
+function Registration() {
+    const [email, setEmail] = useState();
+    const [password, setPassword] = useState();
+    const [firstName, setFirstName] = useState();
+    const [lastName, setLastName] = useState();
+    const navigate = useNavigate();
+    const [error, setError] = useState({firstName: "", lastName: "", email: "", password: ""});
+    const [showPassword, setShowPassword] = useState(false);
+    const [passwordStrengthMessage, setPasswordStrengthMessage] = useState();
+    const [passwordStrength, setPasswordStrength] = useState();
+
+    function createAccount() {
+        personService.createAccount(firstName, lastName, email, password).then((response) => {
+            if (localStorage.getItem('user')) {
+                navigate("/");
+                window.location.reload();
+            }
+        }).catch(error => {
+            if (error.response.data.toLowerCase().includes("email")) {
```

Comment:
Is there any other way to create these message than checking if error message includes email